zendesk_apps_tools.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |s|
   s.summary     = 'Tools to help you develop Zendesk Apps.'
   s.description = s.summary
 
-  s.required_ruby_version = '>= 2.3'
+  s.required_ruby_version = '>= 2.6'
   s.required_rubygems_version = '>= 1.3.6'
```

Comment:
Should we update the required_rubygems_version too?

Reply:
We can, but I'm not sure which version I should bump it to.

Locally I'm using Ruby 2.6.9, and it comes with `gem 3.0.3.1`. So something like `>= 3.0.0`?

Reply:
Pushed 👌